telegram/_bot.py

```diff
@@ -562,14 +568,15 @@ async def send_message(
         disable_notification: DVInput[bool] = DEFAULT_NONE,
         reply_to_message_id: int = None,
         reply_markup: ReplyMarkup = None,
+        allow_sending_without_reply: ODVInput[bool] = DEFAULT_NONE,
+        entities: Union[List["MessageEntity"], Tuple["MessageEntity", ...]] = None,
+        protect_content: ODVInput[bool] = DEFAULT_NONE,
+        *,
```

Comment:
if we have to move these anyway, I think I'd like to put them into the order [defined by telegram](https://core.telegram.org/bots/api#sendmessage). But it's not a priority to me.